src/videouplinkbandwidthpolicy/VideoUplinkBandwidthPolicy.ts

```diff
@@ -57,4 +58,21 @@ export default interface VideoUplinkBandwidthPolicy {
    * Returns the selected [[MediaTrackConstraints]] to update
    */
   chooseMediaTrackConstraints(): MediaTrackConstraints;
+
+  /**
+   * Set a reference to the current transceiver controller.
+   * This will be called during connection and if unified plan is enabled.
+   * This method should not throw.
+   * @param {TransceiverController} transceiverController - The transceiver controller
+   */
+  setTransceiverController?(transceiverController: TransceiverController): void;
+
+  /**
+   * Update the transceiver controller that is set from setTransceiverController such as setEncodingParameters.
+   * Only used when unified plan is enabled but not available for simulcast for now.
+   * The method will be called when a video is enabled or disabled or
+   * when a new bandwidth priority event such as active speaker happens.
```

Comment:
How do I know which? Will it ever matter?

Is it OK for me, as a builder of a policy, to subscribe to `AudioVideoObserver` and `ActiveSpeakerWhatever` and just ignore this method altogether?